src/main/java/mate/controller/driver/DeleteDriverController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet("/cars/drivers/delete")
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+    
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = Long.valueOf(req.getParameter("id"));
+        driverService.delete(driverId);
+        System.out.println("Driver was deleted. Driver ID: " + driverId);
```

Comment:
```suggestion
        resp.sendRedirect("/drivers/all");
```

and same for the rest controllers